src/twisted/newsfragments/12274.bugfix

```diff
@@ -0,0 +1,4 @@
+Tests in twisted.conch.test.test_cftp and twisted.conch.test.test_conch that
+require DSA are now skipped if the installed SSH does not support DSS. The
+availability of DSA is checked by querying `ssh -Q key`, which includes
+`ssh-dss` if DSS is available.
```

Comment:
I think I'd consistently pick one of DSA or DSS except when talking about the protocol-level name.

```suggestion
Tests in twisted.conch.test.test_cftp and twisted.conch.test.test_conch that
require DSA are now skipped if the installed SSH does not support DSA. The
availability of DSA is checked by querying `ssh -Q key`, which includes
`ssh-dss` if DSA is available.
```